velox/common/memory/tests/MemoryManagerTest.cpp

```diff
@@ -591,7 +591,8 @@ TEST_F(MemoryManagerTest, quotaEnforcement) {
     SCOPED_TRACE(testData.debugString());
     std::vector<bool> contiguousAllocations = {false, true};
     for (const auto& contiguousAlloc : contiguousAllocations) {
-      SCOPED_TRACE(fmt::format("contiguousAlloc {}", contiguousAlloc));
+      SCOPED_TRACE(fmt::format(
+          "contiguousAlloc {}", static_cast<bool>(contiguousAlloc)));
```

Comment:
Can use the API `contiguousAlloc.operator bool()` here
https://learn.microsoft.com/en-us/cpp/standard-library/vector-bool-class?view=msvc-170#reference_operator_bool